packages/toolpad-components/src/Text.tsx

```diff
@@ -34,7 +33,21 @@ function Text({ value, markdown, href, loading, mode, sx, ...rest }: TextProps)
       ) : (
         <MarkdownContainer>
           <React.Suspense>
-            <ReactMarkdown remarkPlugins={[remarkGfm]}>{value}</ReactMarkdown>
+            <ReactMarkdown
+              options={{
+                overrides: {
+                  a: {
+                    component: MuiLink,
+                    props: {
+                      target: '_blank',
+                      rel: 'noopener noreferrer nofollow',
```

Comment:
I think that we could let the search engine crawler follow and crawl these links, and be a bit less opinionated https://ahrefs.com/blog/nofollow-links/. If people want to prevent it, they could write a full `<a href` element.

```suggestion
                      rel: 'noopener noreferrer',
```